homeassistant/components/frontend/__init__.py

```diff
@@ -242,7 +241,7 @@ class Panel:
     sidebar_title: str | None = None
 
     # Url to show the panel in the frontend
-    frontend_url_path: str | None = None
+    frontend_url_path: str
```

Comment:
`self.frontend_url_path` will always be set to `str`.
https://github.com/home-assistant/core/blob/427aa55789d172f7cfb9cdf6d6912d5616c34a2b/homeassistant/components/frontend/__init__.py#L256-L270